ARE/middleware/src/main/java/eu/asterics/mw/services/ResourceRegistry.java

```diff
@@ -1113,14 +1114,31 @@ public boolean accept(File dir, String name) {
     public List<URI> getDataList(boolean relative) {
         // get asterics involved data files
         // Not sure how deep we should search, but 10 seems to be enough
-        List<URI> URIs = ComponentUtils.findFiles(toAbsolute(DATA_FOLDER), relative, 10, new FilenameFilter() {
+        List<URI> URIs = ComponentUtils.findFiles(toAbsolute(DATA_FOLDER), relative, RECURSIVE_FILE_SEARCH_DEPTH, new FilenameFilter() {
             @Override
             public boolean accept(File dir, String name) {
                 return true;
             }
         });
         return URIs;
     }
+    
+    /**
+     * Returns a list of URIs of all web 
+     * @param relative
```

Comment:
what does this param mean? please improve comment.

Reply:
ok.
It should state: Returns a list of URIs of resources contained in the Webserver document root or in subpaths.
Currently the search depth is limited to 10 (subfolders)